package.json

```diff
@@ -84,6 +84,7 @@
   },
   "peerDependencies": {
     "@urql/core": ">= 1.11.0",
+    "graphql": ">= 0.11.0",
```

Comment:
```suggestion
    "graphql": "^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0 || ^15.0.0",
```

Reply:
same thing, no? I'd rather support newer versions by default and then make amendments if there are breaking updates (unlikely with the `print` function)